data/data/aws/variables-aws.tf

```diff
@@ -69,3 +69,17 @@ variable "aws_worker_availability_zones" {
   description = "The availability zones to provision for workers.  Worker instances are created by the machine-API operator, but this variable controls their supporting infrastructure (subnets, routing, etc.)."
 }
 
+variable "aws_vpc" {
+  type = string
+  description = "An existing network (VPC ID) into which the cluster should be installed."
```

Comment:
hmm, would this be a required variable? and that would mean we don't support creating a VPC.

Reply:
Set a `null` default and added `(optional)` to the description.